docs/iOS/readme_iOS.md

```diff
@@ -1,69 +1,51 @@
 <a href="/readme.md"><img src="/docs/images/PlayEveryWareLogo.gif" alt="Lobby Screenshot" width="5%"/></a>
 
 <div align="center"> <img src="/docs/images/EOSPluginLogo.png" alt="PlayEveryWare EOS Plugin for Unity" /> </div>
-<br /><br /><br />
 
 ---
 
-
-
 # <div align="center">iOS Getting Started</div> <a name="getting-started" />
 ---
 
-## Prerequisites
-
+## <div align="center">Prerequisites</div>
```

Comment:
I prefer the left aligned headings that were there before. The readme_android.md has left aligned headings too. Having the center aligned main heading "iOS Getting Started" with nothing below it, and then the centered aligned "Prerequisites" looks weird. I think these should be left aligned like the other docs.

Reply:
I tend to agree with you - the decision to keep them centered is reflected in the style guide. There were a variety of opinions on this within the team - I'd be happy to open it up for discussion, but as it exists, with the exception of the README docs which have yet to be substantially reformatted, all the other documents adhere to this pattern.

Reply:
None of the other docs have subheadings center aligned like this though. Some have the main heading center aligned. But none of the other ones have all the subheading center aligned. 

These are just a few examples of docs that were in this PR but are still left aligned. 
![image](https://github.com/PlayEveryWare/eos_plugin_for_unity/assets/50968607/6ef25ed2-7aef-47a8-9939-c9e790126183)
![image](https://github.com/PlayEveryWare/eos_plugin_for_unity/assets/50968607/f035a9be-ddf7-4cbb-b3e0-a3edb6bfb566)
![image](https://github.com/PlayEveryWare/eos_plugin_for_unity/assets/50968607/51a48f8b-89fa-4f81-ae69-a5689c5d94a2)


Here is this file that is center aligned. It stands out as different from everything else. 
![image](https://github.com/PlayEveryWare/eos_plugin_for_unity/assets/50968607/20da99b7-cd0e-4624-8cae-ffad0bc83bcb)

Reply:
I changed directions on this - and actually made all the section headers left-aligned per your suggestion, however I left the document titles centered - and did my best to make sure that this was done consistently across all the documents.

Let me know your thoughts, and "Resolve" this conversation if you think that's appropriate - follow up here otherwise.